spec/provider/testfairy_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'dpl/provider/testfairy'
+
+describe DPL::Provider::TestFairy do
+
+  before (:all) do
+ 
+    %x[mkdir /tmp/android/]
+    ENV['ANDROID_HOME'] = '/tmp/android'
+    %x[echo 'cp $3 $4' > /tmp/android/zipalign]
+    %x[chmod +x /tmp/android/zipalign]
+    
+    @kyestore = '/tmp/debug.keystore'
+    %x[curl -Lso #{@kyestore} http://www.testfairy.com/support-files/travis/dpl/debug.keystore]
+        
+    @local_android_app = '/tmp/android.apk'
+    %x[curl -Lso #{@local_android_app} http://www.testfairy.com/support-files/travis/dpl/android.apk]
+    
+    @local_ios_app = '/tmp/ios.ipa'
+    %x[curl -Lso #{@local_ios_app} http://www.testfairy.com/support-files/travis/dpl/Empty.ipa]
+    
+  end
+  
+  subject :provider do
+    # the account is travis-test@testfairy.com
```

Comment:
In general, specs should not make actual external network calls, because it makes the tests brittle. They will be susceptible to network conditions, and administration of the said service. (If, in the future, you accidentally or intentionally delete `travis-test@testfairy.com` account, our tests will fail without any change in our code or dependencies.)